app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -65,23 +65,23 @@ def ems_inv_to_hashes
     private
 
     def servers
-      @servers ||= @connection.servers_for_accessible_tenants
+      @servers ||= @connection.handled_list(:servers)
```

Comment:
I would personally prefer the direct accessors.  The responsibility of worrying about the handling of a list belongs to the OpenStack handle not to the caller (in this case the parser)